pandas/_libs/tslibs/strptime.pxd

```diff
@@ -1,4 +1,16 @@
+from cpython.datetime cimport (
+    datetime,
+    tzinfo,
+)
 from numpy cimport int64_t
 
 
 cdef bint parse_today_now(str val, int64_t* iresult, bint utc)
+
+
+cdef class ParseState:
```

Comment:
I think we should call this `DateTimeParseState`; helps to disambiguate it in case it ends up intertwined with any other type of parser